Adding additional SCIM instructions (#7480)
dbt-labs/docs.getdbt.com: website/docs/docs/cloud/manage-access/scim.md

1. From the enteprise app **Overview** screen sidebar menu, click **Provisioning**. 
    <Lightbox src="/img/docs/dbt-cloud/access-control/provisioning.png" width="60%" title="The Provisioning option on the sidebar." />
2. From under **Manage**, again click **Provisioning**.
3. Expand the **Mappings** section and click **Provision Microsoft Entra ID users**.
     <Lightbox src="/img/docs/dbt-cloud/access-control/provision-entra-users.png" width="60%" title="Provision the Entra ID users." />
4. Click the box for **Show advanced options** and then click **Edit attribute list for customappsso**.
    <Lightbox src="/img/docs/dbt-cloud/access-control/customappsso-attributes.png" width="60%" title="Click to edit the customappsso attributes." />

> 5. Scroll to the bottom of the **Edit attribute list** window and there will be an empty field where you can add a new entry. Enter the following fields: 

Reviewer: ```suggestion
5. Scroll to the bottom of the **Edit attribute list** window and find an empty field where you can add a new entry with the following fields: 
```

    - **Name:** `emails[type eq "work"].primary`
    - **Type:** `Boolean`
    - **Required:** True
    <Lightbox src="/img/docs/dbt-cloud/access-control/customappsso-entry.png" width="60%" title="Add the new field to the entry list." />
6. Mark all of the fields listed in Step 10 below as `Required`.
    <Lightbox src="/img/docs/dbt-cloud/access-control/mark-as-required.png" width="60%" title="Mark the fields as required." />    
7. Click **Save**

> 8. Back on the **Attribute mapping** screen, click **Add new mapping** and fill out the fields with the following:

Reviewer: We call it "window" above so just suggest being consistent:
```suggestion
8. Back on the **Attribute mapping** window, click **Add new mapping** and complete fields with the following:
```
